app/src/main/java/ai/elimu/appstore/synchronization/AppListActivity.java

```diff
@@ -16,55 +16,53 @@
 import ai.elimu.appstore.R;
 import ai.elimu.appstore.dao.ApplicationDao;
 import ai.elimu.appstore.model.Application;
-import timber.log.Timber;
+import ai.elimu.appstore.util.Const;
 
 public class AppListActivity extends AppCompatActivity {
 
-    private TextView textViewLastSynchronization;
+    private TextView mTextViewLastSynchronization;
```

Comment:
Please don't use `m` in front of property names. We're not following the AOSP guidelines on this.

Reply:
Could you give a reference to that?
Actually as far as I know, the AOSP guidelines, especially for Android, recommends that.
https://source.android.com/source/code-style#follow-field-naming-conventions

Reply:
The reason is that putting an `m` in front of a property name does not make the code easier to read/understand. That's why we are not using it in any of our apps, so it would be best to stay consistent with all the existing Android apps that we have already developed.